### DISCUSSION TITLE
Provide a way to specify path to git

### DISCUSSION THREAD

**Discussion by pgaryali** [gitkraken/vscode-gitlens#1500] [Ideas]
I am using the remote SSH feature in VSCode to connect to a code server. The default git on the remote machine that GitLens picks from /usr/bin/git is 1.8.x. The admins have installed a newer version on an LSF mount and I have no way to tell GitLes which git to use.

Wasn't there a `gitlens.advanced.git` setting that would specify the path to git? What happened to that? Is there any other way I can get this to work? thanks!

**Comment by pgaryali** [comment]
I have updated the `git.path` setting on the remote SSH machine but that does not help GitLens. If I click any of sidebars in GitLens, it fails miserable trying to access `/usr/bin/git` !!

**Reply by eamodio** [reply]
GitLens uses the version of Git that VS Code uses (either via the `git.path` setting or a file system scan). Is the built-in Git extension finding the correct version of Git? You should be able to see it in the `Git` output channel.

**Reply by maksim-pinguin** [reply]
VS Code needs a different path depending on the setup of the integrated terminal. VSCode's build in git seems to ignore the integrated terminal setting

**Comment by fernandoroa** [comment]
necessary to do: https://stackoverflow.com/questions/55988854/alter-path-for-vs-code-remote-ssh
